### PR TITLE
Minor fixes for Install template endpoint

### PR DESCRIPTION
**Pull request by arjuncooliitr**
## Description
Minor fixes for Install template endpoint

## Motivation and Context

While locally testing the Install template endpoint found few issues, so pushing fixes for them.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by MichaelGoberling** [comment]
Perhaps we add some unit tests to cover both uppercase and lowercase cases?

File: actions/templates/install/index.js [hunk 1]
```diff
@@ -149,12 +149,12 @@ async function main (params) {
     const credentialFlowType = credentials[0].flowType;
     let createIntegrationResponse = {};
 
-    if (credentialFlowType === 'adobeid') {
+    if (credentialFlowType.toLowerCase() === 'adobeid') {
```
[the remark this answers is not in the document]
Comment (arjuncooliitr) [reply actions/templates/install/index.js]: Sure good catch. Replaced these with constants.

**Review by shazron — APPROVED** [review]

File: actions/templates/install/index.js [hunk 2]
```diff
@@ -149,12 +150,12 @@ async function main (params) {
...
     let createIntegrationResponse = {};
 
-    if (credentialFlowType === 'adobeid') {
+    if (credentialFlowType.toLowerCase() === CREDENTIAL_FLOW_TYPE_ADOBEID) {
       // form integration request body
       const createAdobeIdIntegrationReqBody = {
         name: String(body.projectName),
         description: String(body.description ? body.description : `Created from template ${template.name}`),
-        platform: String(mapAdobeIdCredentialTypeToPlatformType[credentialType]),
+        platform: String(mapAdobeIdCredentialTypeToPlatformType[credentialType.toLowerCase()]),
```
Comment (purplecabbage) [inline actions/templates/install/index.js:158]: nit: Can this be null? It appears you cover the case where it is not a string.  Either it is always defined, and always a string, or not.

**Review by purplecabbage — COMMENTED** [review]
looks good.  left 1 comment

**Pull request merged**